tools/depends/configure.ac

```diff
@@ -330,18 +321,22 @@ case $host in
 	  platform_cxxflags+=" -fembed-bitcode"
           sdk_name=$target_platform$use_sdk
           platform_min_version="$target_platform-version-min=9.0"
-        else
-          # setup which cpu to use
-          if test "x$use_cpu" = "xauto"; then
-            use_cpu=armv7
+          platform_arch=aarch64
+          if test "x$use_arch" = "xauto"; then
+            use_arch=arm64
           fi
-
+        else
           target_platform=iphoneos
 
           found_sdk_version=[`$use_xcodebuild -showsdks | grep $target_platform | sort | tail -n 1 | awk '{ print $2}'`]
           use_sdk="${use_sdk:-$found_sdk_version}"
           sdk_name=$target_platform$use_sdk
           platform_min_version="$target_platform-version-min=5.1"
+          platform_ldflags+=" -Wl,-segalign,4000"
```

Comment:
change of behavior - those ldflags should only be present when compiling for arm64!